src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageMetadataControl.xaml

```diff
@@ -78,126 +78,160 @@
         Margin="0,8,0,0"
-        FontWeight="Bold"
-        x:Name="_ownersLabel"
-        AutomationProperties.Name="{Binding Text}"
-        Text="{x:Static nuget:Resources.Label_Owners}" />
-      <ItemsControl
-        x:Name="_panelOwners"
-        Grid.Row="1"
-        Grid.Column="1"
-        ItemsSource="{Binding KnownOwnerViewModels}"
-        Visibility="{Binding Path=KnownOwnerViewModels,Converter={StaticResource NullToVisibilityConverter}}"
-        AutomationProperties.LabeledBy="{Binding ElementName=_ownersLabel}"
-        Focusable="False"
-        Margin="8,8,0,0"
-        AlternationCount="{Binding RelativeSource={RelativeSource Self}, Path=Items.Count}">
-        <ItemsControl.ItemsPanel>
-          <ItemsPanelTemplate>
-            <WrapPanel Orientation="Horizontal" />
-          </ItemsPanelTemplate>
-        </ItemsControl.ItemsPanel>
-        <ItemsControl.ItemTemplate>
-          <DataTemplate>
-            <TextBlock TextWrapping="Wrap">
-              <Hyperlink
-                NavigateUri="{Binding Link, Mode=OneWay}"
-                ToolTip="{Binding Link, Mode=OneWay}"
-                Style="{StaticResource HyperlinkStyle}"
-                AutomationProperties.Name="{x:Static nuget:Resources.Hyperlink_Owner}"
-                Command="{x:Static nuget:PackageManagerControlCommands.OpenExternalLink}"
-                CommandParameter="{x:Static nugettel:HyperlinkType.OwnerProfileDetailsPane}">
-                <Run Text="{Binding Name, Mode=OneTime}" />
-              </Hyperlink><TextBlock Margin="0,0,2,0" Text=",">
-                <TextBlock.Visibility>
-                  <MultiBinding Converter="{StaticResource LastItemToVisibilityConverter}">
-                    <Binding RelativeSource="{RelativeSource Mode=TemplatedParent}" Path="(ItemsControl.AlternationIndex)" />
-                    <Binding RelativeSource="{RelativeSource FindAncestor, AncestorType={x:Type ItemsControl}}" Path="Items.Count" />
-                  </MultiBinding>
-                </TextBlock.Visibility>
-              </TextBlock>
+        TextWrapping="Wrap"
+        Text="{Binding Path=PackageMetadata.Description}" />
+
+      <!-- metadata -->
+      <Grid
+    Grid.Row="2"
+    x:Name="_metadataGrid"
+    Margin="0,8">
+        <Grid.RowDefinitions>
+          <RowDefinition />
+          <RowDefinition />
+          <RowDefinition />
+          <RowDefinition />
+          <RowDefinition />
+          <RowDefinition />
+          <RowDefinition />
+          <RowDefinition />
+          <RowDefinition />
+          <RowDefinition />
+        </Grid.RowDefinitions>
+        <Grid.ColumnDefinitions>
+          <ColumnDefinition
+        Width="Auto" />
+          <ColumnDefinition />
+        </Grid.ColumnDefinitions>
+
+        <!-- version -->
+        <TextBlock
+      Visibility="{Binding Path=PackageMetadata.Version,Converter={StaticResource NullToVisibilityConverter}}"
```

Comment:
I think some indentation got moved out of alignment.

Reply:
They did, I'm going to do another PR to fix indentation after this merges. This makes the comparison a little easier to read rn because a whole section needs to be indented even though nothing really changed.